examples/scala/spark-4.0/README.md

```diff
@@ -0,0 +1,230 @@
+# Spark 4.0 ClickHouse Connector Examples
```

Comment:
Lets also mention in README file
If he wants to create a Spark app what the steps that need to be done